cpp/lib/src/FileUtil.cc

```diff
@@ -51,9 +51,11 @@ AutoTempFile::AutoTempFile(const std::string &path_prefix, bool automatically_re
 {
     std::string path_template(path_prefix + "XXXXXX");
     const int fd(::mkstemp(const_cast<char *>(path_template.c_str())));
-    if (fd == -1)
+    if (fd == -1) {
```

Comment:
We don't usually use blocks for single statements.

Reply:
The scope was added to wrap the multi-line statement.